### PR TITLE
2_edit

### PR DESCRIPTION
**Pull request by AnaScia**

### CODE REVIEW

**Comment by AnaScia** [comment]
image

I created a new component called "EditText" for change Item's name

File: src/component/CreateList.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import Table from './Table';
 import '../styles/createList.css';
+
 function CreateList() {
   const [create, setCreate] = useState('');
```
Comment (Alvacampos) [inline src/component/CreateList.jsx:6]: if it is a list of items, why does the use state start with `''`?. useState can start with all types of data like [], {}, etc.

File: src/component/EditText.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <>
+      <input
+        type="text"
+        name="edit"
+        value={create}
+        onChange={handleClick}
+      ></input>
+      <button
+      // onClick={() => setSave([...save, create])}
```
Comment (Alvacampos) [inline src/component/EditText.jsx]: Should this be commented?

File: src/component/Table.jsx [hunk 3]
```diff
@@ -54,7 +59,7 @@ function Table({ save, setSave }) {
           ))}
         </tbody>
 
-        {/* {modal && <component></component>} */}
+        {/* {modal && <EditText />} */}
```
Comment (Alvacampos) [inline src/component/Table.jsx]: Clean this.

File: src/component/CreateList.jsx (hunk 1 above)
Comment (AnaScia) [reply src/component/CreateList.jsx:6]: because it is for input.Instead "save" is a list of items

Comment (AnaScia) [reply src/component/CreateList.jsx:6]: Is it wrong?

File: src/component/EditText.jsx (hunk 2 above)
Comment (AnaScia) [reply src/component/EditText.jsx]: sorry,I forgot to delete this.I was trying something

**Comment by AnaScia** [comment]
BEFORE EDIT:
image

AFTER EDIT:
image

**Pull request merged**